src/Uno.UI/UI/Xaml/ResourceDictionary.cs

```diff
@@ -159,12 +159,12 @@ private void TryMaterializeLazy(object key, ref object value)
 			}
 		}
 
-		private bool GetFromMerged(object key, out object value)
+		private bool GetFromMerged(object key, out object value, ElementTheme elementTheme = ElementTheme.Default)
```

Comment:
I would assume that this parameter should not be optional, or at least, it should default to the `RequestedTheme` (same thing for all other methods that now accepts a theme as parameter).

Reply:
+1, I prefer not to have default parameter values on private or internal methods, unless it'd really introduce a lot of noise otherwise.